utils/dd4hep2root

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+### Script to generate ROOT file with detector geometry from detector xml file, 
+### from https://fccsw.web.cern.ch/fccsw/tutorials/static/python/dd4hep2root
+### usage described in FCC software tutorial: 
+### https://hep-fcc.github.io/fcc-tutorials/master/full-detector-simulations/Visualization/Visualization.html#detector-geometry
+
+import sys
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Convert detector')
+    parser.add_argument('-c', '--compact', help='Compact file location(s)',
+                        required=True, type=str, nargs='+')
+    parser.add_argument('-o', '--out', help='Converted file path',
+                        default='detector.root', type=str)
+    args = vars(parser.parse_args())
+
+    convert(args['compact'], args['out'])
```

Comment:
```suggestion
    args = parser.parse_args()

    convert(args.compact, args.out)
```

That's how this should work, isn't it?

Reply:
Yep, it also worked before though, could propagate it to https://hep-fcc.github.io/fcc-tutorials/master/full-detector-simulations/Visualization/Visualization.html#detector-geometry as well

Reply:
I don't see any `vars` at the link you posted?